package.json

```diff
@@ -21,6 +21,11 @@
   "engines": {
     "node": ">=12.2.0"
   },
+  "main": "./src/loader.js",
+  "bin": {
+    "presm": "./src/loader.js",
```

Comment:
Is this actually used? I would expect the loader and the on-the-fly command to be in two separate files.